src/citrine/resources/table_config.py

```diff
@@ -349,8 +349,9 @@ def default_for_material(
         ambiguous = [(Variable.build(v), Column.build(c)) for v, c in data['ambiguous']]
         return config, ambiguous
 
-    def preview(self, *,
+    def preview(self,
                 table_config: TableConfig,
+                *,
```

Comment:
Once the deprecated `preview_roots` is removed, then this method will have two necessary and equally-important arguments, so they should both be kwargs